benchalerts/tests/unit_tests/expected_md/details_nocommit.md

```diff
@@ -0,0 +1,3 @@
+This report was not associated with any commit connected to the git graph. This probably means that benchmarks were run on a transient merge-commit.
```

Comment:
@jonkeane To address your latest comment, I added this sentence to the details. Those show up at the bottom of a report (like [this one](https://github.com/conbench/benchalerts/runs/13049239396))

Reply:
This is fantastic, thank you!